plenum/server/monitor.py

```diff
@@ -26,6 +26,49 @@
 logger = getlogger()
 
 
+class RequestMeasurement:
```

Comment:
Do we need to re-calculate the throughput again in NotifierPlugingManager to detect suspicious spikes?